src/bench/gcs_filter.cpp

```diff
@@ -15,29 +15,56 @@ static void ConstructGCSFilter(benchmark::Bench& bench)
         elements.insert(std::move(element));
     }
 
+    return elements;
+}
+
+static void GCSBlockFilterGetHash(benchmark::Bench& bench)
```

Comment:
Could name it GCSFilterGetHash instead of GCSBlockFilterGetHash for consistency with the other benchmarks.